decidim/decidim
# Rename "terms and conditions" to "terms of service"

#### :tophat: What? Why?
As discussed with the maintainers, we want to standardize the naming for "terms of service".

We are using a few different variations of this and it was decided that we will use the "terms of service" everywhere.

#### :pushpin: Related Issues
- Related to #10566

#### Testing
- Try the `/pages/terms-and-conditions` path and see that it redirects you to `/pages/terms-of-service`
- Create a new organization and see that the page slug is `/pages/terms-of-service`
- Update the admin user's accepted TOS version (e.g. `Decidim::User.first.update!(accepted_tos_version: nil)`) and see that the user can accept the terms
